packages/enzyme/src/Debug.js

```diff
@@ -66,6 +64,7 @@ function indentChildren(childrenStrs, indentLength) {
 
 export function debugNode(node, indentLength = 2, options = {}) {
   if (typeof node === 'string' || typeof node === 'number') return escape(node);
+  if (typeof node === 'function') return '[function]';
```

Comment:
a) this should probably be `[function Foo]` where Foo is the functionName
b) could this small change be peeled out into a separate PR with its own tests?